packages/server/src/comment/comment.resolver.ts

```diff
@@ -6,18 +6,20 @@ import { UseGuards } from '@nestjs/common';
 import { JwtAuthGuard } from 'src/auth/jwt.guard';
 import { UserContext } from 'src/auth/user.decorator';
 import { TokenPayload } from 'src/auth/user.dto';
+import { FileService } from 'src/file/file.service';
+import { File } from 'src/file/file.model';
 
 @Resolver(() => Comment)
 @UseGuards(JwtAuthGuard)
 export class CommentResolver {
-  constructor(private commentService: CommentService) {}
+  constructor(private commentService: CommentService, private fileService: FileService) {}
 
   @Mutation(() => Comment)
   async addComment(@UserContext() user: TokenPayload, @Args('input') input: CreateCommentInput) {
```

Comment:
NIT: On all the methods we should have a return type. So `Promise<Comment>` here